src/internal_events/kafka.rs

```diff
@@ -4,48 +4,82 @@ use metrics::{counter, gauge};
 use vector_core::{internal_event::InternalEvent, update_counter};
 
 #[derive(Debug)]
-pub struct KafkaEventReceived {
+pub struct KafkaEventsReceived {
     pub byte_size: usize,
+    pub count: usize,
 }
```

Comment:
Is this the same as the generic `EventsReceived`, or are there deprecated outputs that need preserving?

Reply:
There are some deprecated events that we need to preserve 🤷